documentation/sdk/docs/how_tos/accounts_and_addresses/create_account.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Create an Account
+description: 'How to create an account to start sending value transactions to the Tangle.'
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- create account
+- create wallet
+- account manager
+- multi account approach
+- single account approach
+- nodejs
+- python
+- java
+- rust
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import RustCode from "!!raw-loader!../../../../../sdk/examples/how_tos/accounts_and_addresses/create_account.rs";
+import NodejsCode from "!!raw-loader!../../../../../bindings/nodejs/examples/how_tos/accounts_and_addresses/create_account.ts";
+import PythonCode from "!!raw-loader!../../../../../bindings/python/examples/how_tos/accounts_and_addresses/create_account.py";
+import TipAccountApproaches from "../../_admonitions/_tip_account_approaches.md";
+
+The first thing you will need to do when you are getting started with the SDK is to create an account.
```

Comment:
I'm also not a TW, but here's my suggestion :trollface: 
```suggestion
```

Reply:
```suggestion
This How-To will show you how to create an account.
```

I improved it :trollface: